src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -167,7 +167,21 @@
                         Content="{x:Static local:AdvancedOptionPageStrings.Option_Colorize_regular_expressions}" />
                     <CheckBox x:Name="Report_invalid_regular_expressions"
                         Content="{x:Static local:AdvancedOptionPageStrings.Option_Report_invalid_regular_expressions}" />
-                    <CheckBox x:Name="Highlight_related_components_under_cursor"
+                    <CheckBox x:Name="Highlight_related_regular_expression_components_under_cursor"
+                        Content="{x:Static local:AdvancedOptionPageStrings.Option_Highlight_related_components_under_cursor}" />
+                </StackPanel>
+            </GroupBox>
+
+            <GroupBox x:Uid="JsonGroupBox"
```

Comment:
Did we update the search keywords for our pages for both C# and VB? (@ryzngard look I remembered this once in code review!)

Reply:
no clue how to do that.  @ryzngard ?

Reply:
Step number 4: https://github.com/ryzngard/roslyn/blob/main/src/VisualStudio/CSharp/Impl/Options/Readme.md

Reply:
Thanks!